_source/logzio_collections/_log-sources/azure-graph.md

```diff
@@ -0,0 +1,133 @@
+| type | The type of the OAuth API. Currently we support the following types: azure_graph, general. | Required | 
+| name | The name of the OAuth API. Please make names unique. | Required | 
+| credentials.id | The OAuth API credentials id. | Required | 
+| credentials.key | The OAuth API credentials key. | Required |
+| http_request.method | The HTTP method. Can be GET or POST. | Required | 
+| http_request.url | The OAuth API url. Make sure the url is without `?` at the end. | Required | 
+| http_request.headers | Pairs of key and value the represents the headers of the HTTP request. | Optional | 
+| http_request.body | The body of the HTTP request. Will be added to HTTP POST requests only. | Optional | 
+| token_http_request.method | The HTTP method. Can be GET or POST. | Required |
+| token_http_request.url | The OAuth API token request  url. Make sure the url is without `?` at the end. | Required | 
+| token_http_request.headers | Pairs of key and value the represents the headers of the HTTP request. | Optional |
+| token_http_request.body | The body of the HTTP request. Will be added to HTTP POST requests only. | Optional | 
+| json_paths.data_date | The json path to the data's date value inside the response of the OAuth API. | Required | 
+| settings.time_interval | The OAuth API time interval between runs. | Required | 
+| settings.days_back_fetch | The max days back to fetch from the OAuth API. | Optional. Default value is 14 days.| 
+| filters | Pairs of key and value of parameters that can be added to the OAuth API url. Make sure the keys and values are valid for the OAuth API. | Optional |
+| custom_fields | Pairs of key and value that will be added to each data and be sent to Logz.io. | Optional | 
+
+##### Create a Last Start Dates text file
+
+Create an empty text file named last_start_dates.txt in the same directory as the config file:
+
+```shell
+$ touch last_start_dates.txt
+```
+
+After every successful iteration of an API, the last start date of the next iteration will be written to last_start_dates.txt. Each line starts with the API name and ends with the last start date.
+
+If you stopped the container, you can continue from the exact place you stopped, by adding the date to the API filters in the configuration.
+
+##### Run the Docker container
+
+```shell
+docker run --name logzio-api-fetcher \
+-v "$(pwd)":/app/src/shared \
+logzio/logzio-api-fetcher
+```
+
+##### Stop the Docker container
+
+When you stop the container, the code will run until the iteration is completed. To make sure it will finish the iteration on time, please give it a grace period of 30 seconds when you run the `docker stop` command.
+
+```shell
+docker stop -t 30 logzio-api-fetcher
+```
+
+##### Check Logz.io for your logs
+
+Give your logs some time to get from your system to ours,
+and then open [Kibana](https://app.logz.io/#/dashboard/kibana). You can filter for data of type `azure_graph` to see the incoming Microsoft Graph logs.
```

Comment:
Can you please change the last part of the line to this:
You can filter for data of your custom field type value or type `api_fetcher` otherwise, to see the incoming Microsoft Graph logs.